cmd/manager/main.go

```diff
@@ -160,7 +155,7 @@ func main() {
 	}
 
 	// Add the Metrics Service
-	addMetrics(ctx, cfg)
+	// addMetrics(ctx, cfg)
```

Comment:
what is the plan around this?

Reply:
I should have simply deleted it.
According to openshift docs, we should us InstrumentedEnqueueRequestForObject, and delete addMetrics:
https://sdk.operatorframework.io/docs/building-operators/golang/migration/